src/views/AccessControl.tsx

```diff
@@ -325,13 +334,26 @@ export const AccessControl = () => {
                                             setCurrentPage(page)
                                         }
                                     }}
-                                    className={`${showTutorial ? "card-table card-table-no-placeholder" : "card-table"}`}
+                                    className={`access-control-table ${showTutorial ? "card-table card-table-no-placeholder" : "card-table"}`}
```

Comment:
Codacy has a fix for the issue: [Replace `····································className={`access-control-table·${showTutorial·?·"card-table·card-table-no-placeholder"·:·"card-table"}`}` with `··················className={`access-control-table·${⏎····················showTutorial⏎······················?·"card-table·card-table-no-placeholder"⏎······················:·"card-table"`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
                  className={`access-control-table ${
```